pysatNASA/instruments/ses14_gold.py

```diff
@@ -189,10 +189,18 @@ def load(fnames, tag=None, inst_id=None):
                         [dt.datetime.strptime(str(val), "b'%Y-%m-%dT%H:%M:%SZ'")
                          for val in data['scan_start_time'].values])
         data = data.swap_dims({'nscans': 'time'})
+        meta['time'] = meta['scan_start_time']
+        meta['time']['long_name'] = 'time'
+        meta['time']['notes'] = 'Converted to datetime from scan_start_time'
+
+        # Update coordinates with dimensional data
         data = data.assign_coords({'nlats': data['nlats'],
                                    'nlons': data['nlons'],
                                    'nmask': data['nmask'],
                                    'channel': data['channel'],
                                    'hemisphere': data['hemisphere']})
+        meta['nlats'] = {'notes': 'Index for latitude values'}
+        meta['nlons'] = {'notes': 'Index for longitude values'}
+        meta['nmask'] = {'notes': 'Index for mask values'}
```

Comment:
@rstoneback, is there a way of future-proofing this using existing pysat.Meta syntax?  Basically add the 'notes' info regardless of whether this meta exists?

Reply:
These commands should work independent of existing information for 'nlats' etc

Reply:
But you should use something like `inst.meta.labels.notes` instead of `'notes'`

Reply:
OK, verified that this will work if meta already exists.